Sources/Controllers/TravelGuides/article/TravelArticleDialogViewController.swift

```diff
@@ -146,71 +146,78 @@ class TravelArticleDialogViewController : OABaseWebViewController, TravelArticle
-        bottomStackView!.addArrangedSubview(contentButton!)
+        guard let contentButton else {return}
+        contentButton.setImage(UIImage(named: "ic_custom_list"), for: .normal)
+        contentButton.tintColor = UIColor.iconColorActive
+        contentButton.contentHorizontalAlignment = .left
+        contentButton.addTarget(self, action: #selector(self.onContentsButtonClicked), for: .touchUpInside)
+        bottomStackView.addArrangedSubview(contentButton)
         
-        bottomStackView!.addArrangedSubview(UIView())
+        bottomStackView.addArrangedSubview(UIView())
         
         pointsButton = UIButton()
-        pointsButton!.setTitle(localizedString("shared_string_gpx_points"), for: .normal)
-        pointsButton!.setTitleColor(UIColor.textColorActive, for: .normal)
-        pointsButton!.titleLabel?.font = UIFont.preferredFont(forTextStyle: .body)
-        pointsButton!.addTarget(self, action: #selector(self.onPointsButtonClicked), for: .touchUpInside)
-        bottomStackView!.addArrangedSubview(pointsButton!)
+        guard let pointsButton else {return}
+        pointsButton.setTitle(localizedString("shared_string_gpx_points"), for: .normal)
+        pointsButton.setTitleColor(UIColor.textColorActive, for: .normal)
+        pointsButton.titleLabel?.font = UIFont.preferredFont(forTextStyle: .body)
+        pointsButton.addTarget(self, action: #selector(self.onPointsButtonClicked), for: .touchUpInside)
+        bottomStackView.addArrangedSubview(pointsButton)
         
-        bottomStackView!.addArrangedSubview(UIView())
+        bottomStackView.addArrangedSubview(UIView())
         
         bookmarkButton = UIButton()
-        bookmarkButton!.setImage(UIImage.templateImageNamed("ic_navbar_bookmark_outlined"), for: .normal)
-        bookmarkButton!.tintColor = UIColor.iconColorActive
-        contentButton!.contentHorizontalAlignment = .right
-        bookmarkButton!.addTarget(self, action: #selector(self.onBookmarkButtonClicked), for: .touchUpInside)
-        bottomStackView!.addArrangedSubview(bookmarkButton!)
+        guard let bookmarkButton else {return}
+        bookmarkButton.setImage(UIImage(named:"ic_navbar_bookmark_outlined"), for: .normal)
+        bookmarkButton.tintColor = UIColor.iconColorActive
+        contentButton.contentHorizontalAlignment = .right
+        bookmarkButton.addTarget(self, action: #selector(self.onBookmarkButtonClicked), for: .touchUpInside)
+        bottomStackView.addArrangedSubview(bookmarkButton)
         updateBookmarkButton()
     }
     
     func updateBookmarkButton() {
         if let article {
-            let isSaved = TravelObfHelper.shared.getBookmarksHelper().isArticleSaved(article: article)
-            let iconName = isSaved ? "ic_navbar_bookmark" : "ic_navbar_bookmark_outlined"
-            bookmarkButton!.setImage(UIImage.templateImageNamed(iconName), for: .normal)
+            if let bookmarkButton {
```

Comment:
If the check is done at the beginning of the function, it's better to use the 'guard let' construct